docs/user-guide/node-selection/pod-with-node-affinity.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
It's kind of weird to have an example without the complete text documentation to go along with it.  Fine for a follow PR if needed.